inductiva/fluids/dam_break/README.md

```diff
@@ -0,0 +1,39 @@
+# Dam Break
+
+This scenario simulates the motion of a fluid in a cubic tank only subject to gravity.
+The simulation is performed using the [Smoothed Particle Hydrodynamics](https://en.wikipedia.org/wiki/Smoothed-particle_hydrodynamics)
+method. Currently, this scenario runs with both [SPlisHSPlasH](https://github.com/inductiva/inductiva/blob/main/inductiva/simulators/splishsplash.py) and [DualSPHysics](https://github.com/inductiva/inductiva/blob/main/inductiva/simulators/dualsphysics.py).
+
+This scenario allows users to select from different fluids - Water, Honey, Olive oil, Beer, Jet fuel and a few more (see [here](https://github.com/inductiva/inductiva/blob/main/inductiva/fluids/fluid_types.py) further types) - the dimensions of the cube and its initial position.
```

Comment:
In this scenario, users can pick different fluids (like Water, Honey, Olive oil, Beer, Jet fuel, and more, listed [here](https://github.com/inductiva/inductiva/blob/main/inductiva/fluids/fluid_types.py)), adjust the cube's size, and set its initial position.